diff -Nur old/doc/clamsmtpd.conf new/doc/clamsmtpd.conf
--- old/doc/clamsmtpd.conf.orig	2005-03-12 06:28:08.000000000 +0100
+++ old/doc/clamsmtpd.conf	2005-05-13 01:09:28.000000000 +0200
@@ -29,13 +29,13 @@
 #Listen: 0.0.0.0:10025
 
 # The address clamd is listening on
-#ClamAddress: /var/run/clamav/clamd
+#ClamAddress: /var/lib/clamav/clamd.socket
 
 # A header to add to all scanned email
 #Header: X-Virus-Scanned: ClamAV using ClamSMTP
 
 # Directory for temporary files
-#TempDirectory: /tmp
+#TempDirectory: /var/spool/clamsmtpd/tmp
 
 # Whether or not to bounce email (default is to silently drop)
 #Bounce: off
